src/main/java/GPT/Deadline.java

```diff
@@ -0,0 +1,56 @@
+package GPT;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task with a deadline in the GPT application.
+ * This class extends the Task class to include a specific date and time by which the task should be completed.
+ */
+class Deadline extends Task {
+    private LocalDateTime by;
+    private static final DateTimeFormatter OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("MMM dd yyyy, h:mma");
+
+    /**
+     * Constructs a Deadline task with the specified description and deadline.
+     *
+     * @param description The description of the task.
+     * @param by          The date and time by which the task should be completed.
```

Comment:
Thank you for the positive feedback! I appreciate your suggestion regarding the alignment. I initially aligned the parameters for consistency and readability, but I agree that it might not be necessary to be this precise. I will simplify the spacing to make it less rigid while keeping the documentation clear.